### ISSUE TITLE
Remove template code duplication

### ISSUE THREAD

**Issue by Kixiron** [rust-lang/docs.rs#955]
There's some code duplication in templates that's really annoying me

[`crate/details.html`](https://github.com/rust-lang/docs.rs/blob/master/templates/crate/details.html#L14-L115)=> [`rustdoc/body.html`](https://github.com/rust-lang/docs.rs/blob/master/templates/rustdoc/body.html#L53-L118)

**Comment by giraffate** [comment]
Hi, is the above links broken? Especially in the latter, `#L53-L118` is not found now.

**Comment by jyn514** [comment]
Yes, that part of the codebase had a lot of churn lately. Here's an updated link for `rustdoc/header.html`: https://github.com/rust-lang/docs.rs/blob/master/templates/rustdoc/header.html#L52-L117

**Comment by giraffate** [comment]
Thanks for your advice!
